subsys/net/lib/ocpp/ocpp.c

```diff
@@ -287,15 +290,18 @@ static void ocpp_internal_handler(void *p1, void *p2, void *p3)
 			break;
 
 		case PDU_REMOTE_START_TRANSACTION:
-			ctx->cb(OCPP_USR_START_CHARGING, &msg.usr, ctx->user_data);
+			msg.msgtype = (enum ocpp_pdu_msg)OCPP_USR_START_CHARGING;
```

Comment:
why are you reintroducing this? this issue was already fixed..

Reply:
changes is on top of it.

Reply:
I didn't see that issue addressed in the description from the other PR, so assumed it was not. I can drop that commit here.

Reply:
@aescolar I have mistaken clang warning still after https://github.com/zephyrproject-rtos/zephyr/commit/dece77924f7c415691b1112ac38fc796d05dd640